### PR TITLE
Add index of videos and newsletter topics

### PR DESCRIPTION
**Pull request by bethaitman**
A page to make all our talk videos and newsletter articles more discoverable.

Yes, this is currently a hand-written monstrosity, but I think it's still worth having.

### CODE REVIEW

**Comment by bethaitman** [comment]
Note that I haven't done recent newsletter topics (I think I have quite a few months of backlog to catch up on), nor Prague 2019. I can do those in a later PR.

File: docs/about/learning-resources.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Learning Resources
    :maxdepth: 2
    :hidden:
 
-   /newsletter
```
Comment (ericholscher) [inline docs/about/learning-resources.rst:9]: We should continue listing the newsletter here, I think

**Review by ericholscher — APPROVED** [review]
I'm a huge fan of this. We can grow it from here, but it's already a huge upgrade. I say we ship it and iterate. 🎉

File: docs/topics.rst [hunk 2]
```diff
@@ -0,0 +1,558 @@
+All our videos and articles
+=============================
+
+This page links to the topics that have been covered by `conference </conf/>`__ talks or in the `newsletter </newsletter/>`__. They’re in no particular order, and some topics are repeated if they fit into more than one category. Enjoy!
+
+If you think something is wrongly categorised, please send us a pull request :)
+
```
Comment (plaindocs) [inline docs/topics.rst:7]:
```suggestion
.. contents::
   :local:
   :depth: 2
   :backlinks: none
   
```
Looks like, and I find it helpful to parse the amount of info here. @bethaitman what do you think?

image

**Review by plaindocs — APPROVED** [review]
@bethaitman this is awesome! Let's merge now and iterate later.

File: docs/about/learning-resources.rst (hunk 1 above)
Comment (bethaitman) [reply docs/about/learning-resources.rst:9]: I can add it back in, but it's slightly weird as that's the "Newsletter and mailing lists" page. I think it'd make more sense to link to the newsletter archive here, but I'm not sure how to do that.

File: docs/topics.rst (hunk 2 above)
Comment (bethaitman) [reply docs/topics.rst:7]: Agreed, I think this is super helpful!

**Comment by plaindocs** [comment]
The vale installer is being silly - I'll poke.

**Comment by bethaitman** [comment]
@plaindocs - is something wrong? as far as I can tell this is okay to merge now.

**Comment by ericholscher** [comment]
Shipping it. 🚢

File: docs/about/learning-resources.rst (hunk 1 above)
Comment (ericholscher) [reply docs/about/learning-resources.rst:9]: Ah, it's still in the Stay Connected section -- ignore me :)

**Comment by plaindocs** [comment]
CI failed, but it sorted itself out when I kicked it. Sorry for not being clearer.

**Pull request merged**